pkgs/tools/typesetting/tex/texlive/combine.nix

```diff
@@ -224,25 +242,26 @@ in (buildEnv {
     # sort entries to improve reproducibility
     [[ -f "$TEXMFSYSCONFIG"/web2c/updmap.cfg ]] && sort -o "$TEXMFSYSCONFIG"/web2c/updmap.cfg "$TEXMFSYSCONFIG"/web2c/updmap.cfg
 
-    perl "$out"/share/texmf/scripts/texlive/mktexlsr.pl --sort "$out"/share/texmf-* # to make sure
+    perl "$TEXMFDIST"/scripts/texlive/mktexlsr.pl --sort "$TEXMFSYSCONFIG" "$TEXMFSYSVAR" # to make sure
   '' +
     # install (wrappers for) scripts, based on a list from upstream texlive
   ''
     source '${bin.core.out}/share/texmf-dist/scripts/texlive/scripts.lst'
     for s in $texmf_scripts; do
-      [[ -x "$out/share/texmf/scripts/$s" ]] || continue
+      [[ -x "$TEXMFDIST/scripts/$s" ]] || continue
       tName="$(basename $s | sed 's/\.[a-z]\+$//')" # remove extension
       [[ ! -e "$out/bin/$tName" ]] || continue
-      ln -sv "$(realpath $out/share/texmf/scripts/$s)" "$out/bin/$tName" # wrapped below
+      ln -sv "$(realpath "$TEXMFDIST/scripts/$s")" "$out/bin/$tName" # wrapped below
```

Comment:
This line could be moved to `${texmf}` now?

Reply:
I suppose I'd prefer to leave the `bin` folder responsibility to the derivation that also does the shebangs and the wrapping. At least for now, because I have something else in mind for the script (in short: why not generate & patch the `bin` link right when creating the package, with the appropriate care not to destroy the fixed outputs and so on – and combine does wrapping only).

Reply:
Note for the future: generate `bin` script links early on. Best scenario, generate the `patchSheBangs`'d scripts right when building the package itself, so that `texlive.combine` is only in charge of the wrappers.